workflows/trana.nf

```diff
@@ -94,18 +94,27 @@ workflow TRANA {
                 meta, reads -> [meta, [], reads]
             }).reads.set { ch_processed_reads }
 
+            // Split channels into read files with or without read content
+            ch_processed_reads
+            .branch { it ->
+                passed: it[1].size() > 20
+                failed: it[1].size() <= 20
```

Comment:
So, this means less than 20 reads and it will fail, or reads 20 bp or shorter?

Reply:
20 bytes because gzipped empty file is always 20 bytes as a result of the header